devai-cli/src/devai/util/constants.py

```diff
@@ -0,0 +1,4 @@
+"""Constants used throughout the DevAI CLI."""
```

Comment:
already have config - https://github.com/GoogleCloudPlatform/genai-for-developers/blob/main/devai-cli/src/devai/commands/constants.py#L2